### ISSUE TITLE
iOS Voice control cannot undo or redo text editing

### ISSUE THREAD

**Issue by chunhtai** [flutter/flutter#77614]
```dart
import 'package:flutter/cupertino.dart';
import 'package:flutter/material.dart';
void main() {
  runApp(
    MaterialApp(
      home: Scaffold(
        appBar: AppBar(
          title: Text('Expansion Tile Splash'),
          centerTitle: true,
        ),
        body: TextField(),
      ),
    ),
  );
}
```

1. in iOS, turn on voice control
2. tap text field to start editing
3. type "how are you" to the text field
4. speak "undo"
5. speak "redo"

expect: the undo should remove the text in the textfield, and the redo should retype in the text back to the textfield.
actual: nothing happened.

**Comment by chunhtai** [comment]
I couldn't figure out what the iOS was trying to do when it called the selector of FlutterTextInputView. I am not sure how to proceed at this point.

**Comment by chunhtai** [comment]
There are two different direction we can go:

1. We know UITextField is working natively, if we can somehow convert our TextInputPlugin which is a UITextInput subclass to UITextField subclass. We should be able to make it work
2. Build a native app that uses UITextInput that can work with the redo and undo, and figure out what is missing on our TextInputPlugin. Right now, I can't find a comprehensive example of UITextInput that can work with voice control. I will need to keep trying.

**Comment by chunhtai** [comment]
looks like the undomanager is working for voice control as well closing
